prog/minio/minio_server_nexus.rb

```diff
@@ -41,7 +41,7 @@ def self.assemble(minio_pool_id, index)
 
   def before_run
     when_destroy_set? do
-      if strand.label != "destroy"
+      if !destroying_set?
```

Comment:
Up to you, but this PR mixes `if !destroying_set?` and `unless destroying_set?`.

Reply:
While refactoring, I used `unless destroying_set?` for everything, but since this condition tree has `elsif`, we can't mix `unless` and `elsif`. So, I changed them to `if !*_set?`. Since `hop_*` raises an exception, there's no need for `elsif`; we can just use `if` if you prefer.